sensor-core/src/main/java/com/mindlamp/Accelerometer.java

```diff
@@ -6,9 +6,6 @@
 import android.content.Context;
 import android.content.Intent;
```

Comment:
The package name should be `digital.lamp.____` and not `com.mindlamp.___`. Please rename/fix the references appropriately.

Reply:
Done